tests/integration/disambiguation/test_disambiguation_api.py

```diff
@@ -43,26 +43,34 @@
 '''
 
 
-def test_save_signatures_and_clusters(isolated_app, tmpdir):
+def test_save_curated_signatures_and_input_clusters(isolated_app, tmpdir):
     TestRecordMetadata.create_from_file(__name__, '792017.json')
 
-    clusters_fd = tmpdir.join('clusters.json')
-    signatures_fd = tmpdir.join('signatures.jl')
+    curated_signatures_fd = tmpdir.join('curated_signatures.jsonl')
+    input_clusters_fd = tmpdir.join('input_clusters.jsonl')
 
     config = {
-        'DISAMBIGUATION_CLUSTERS_PATH': str(clusters_fd),
-        'DISAMBIGUATION_SIGNATURES_PATH': str(signatures_fd),
+        'DISAMBIGUATION_CURATED_SIGNATURES_PATH': str(curated_signatures_fd),
+        'DISAMBIGUATION_INPUT_CLUSTERS_PATH': str(input_clusters_fd),
     }
 
     with patch.dict(current_app.config, config):
-        save_signatures_and_clusters()
+        save_curated_signatures_and_input_clusters()
 
-    clusters = json.load(clusters_fd)
+    input_clusters = [json.loads(line) for line in input_clusters_fd.readlines()]
+    reversed_input_clusters = {
+        input_cluster['cluster_id']: {
+            'author_id': input_cluster['author_id'],
+            'signature_uuids': input_cluster['signature_uuids'],
+        } for input_cluster in input_clusters
+    }  # XXX: so that the assertion doesn't depend on cluster_id.
 
-    assert '1010819' in clusters
-    assert ['94f560d2-6791-43ec-a379-d3dc4ad0ceb7'] == clusters['1010819']
+    assert {
+        'author_id': 1010819,
+        'signature_uuids': ['94f560d2-6791-43ec-a379-d3dc4ad0ceb7'],
+    } in reversed_input_clusters.values()
```

Comment:
why not simply `assert any(cluster for cluster in input_cluster if 'author_id' == 1010819 and 'signature_uuids == ['94f560d2-6791-43ec-a379-d3dc4ad0ceb7'])`?

Reply:
I actually prefer my assertion here because I don't think `py.test` would be able to provide any useful error message when presented with an `any`... Also, if you `--pdb` in my assertion when it fails you have some state (the `reversed_input_clusters` variable) to inspect.

Reply:
if you `--pdb` you can inspect `input_clusters` in any case, but OK.